jira.js

```diff
@@ -59,8 +60,8 @@ function printMeta(){
   xhr.open("GET", "https://yexttest.atlassian.net/rest/api/2/issue/createmeta", false);
 
   xhr.onreadystatechange=function() {
-  if (xhr.readyState === 4){   //if complete
-      if(xhr.status === 200){  //check if "OK" (200)
+    if (xhr.readyState === 4){   //if complete
```

Comment:
this code looks pretty similar to the above, any way we can make it generic?

Reply:
seems like its repeated in a few different places as well

Reply:
Simplified slightly in following PR, will work on making it more generic there.